bin/ci

```diff
@@ -73,6 +73,10 @@ prepare_system() {
 }
 
 build() {
+  # Set git config for doc generator specs
+  with_build_env 'git config --global user.email "you@example.com"'
+  with_build_env 'git config --global user.name "Your Name"'
```

Comment:
If we shell out to `git` in the specs, we should support all git setups without environment tweaks.

`git -c user.email="" -c user.name="" commit ...` is a more robust way to solve this.

Reply:
I just copied what shards does. But your solution looks better, yes.